arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -3705,6 +3705,32 @@
       },
       "description": "The instance view of a virtual machine boot diagnostics."
     },
+    "VirtualMachineIdentity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal id of virtual machine identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant id associated with the virtual machine."
+        },
+        "type": {
+          "type": "string",
+          "description": "The identity type.",
+          "enum": [
```

Comment:
- There is currently only one value for this property, although there may be more in the future such as binding an existing user created identity to a resource.
- The value is provided by the user. By providing this property, the customer is opting into creation of an AAD identity that is associated and available to the resource
- It indicates that the user wants a managed AAD identity to be created for the resource via a new managed secure identity service. This identity can allow access to KeyVault, and other resources. Identity is a generic ARM level concept that can be applied to a resource. The compute's VM and VMSS resources just happen to be some of the first to make it available.